src/project-1.js

```diff
@@ -3,137 +3,165 @@
 const multiplyByTen = (num) => {
```

Comment:
This is fine but you can also simplify the whole function to look like this:
`const multiplyByTen = (num) => num * 10;`
The value is automatically returned.